lib/msf/core/exploit/file_dropper.rb

```diff
@@ -5,16 +5,19 @@ module Exploit::FileDropper
 
   def initialize(info = {})
     super(
-      'Compat' => {
-        'Meterpreter' => {
-          'Commands' => %w[
-            stdapi_fs_delete_dir
-            stdapi_fs_delete_file
-            stdapi_fs_getwd
-            stdapi_fs_stat
-          ]
+      update_info(
```

Comment:
This part's important, as it merges in the module's actual required metadata. Missed by mistake as part of https://github.com/rapid7/metasploit-framework/pull/15609/files#diff-a5e083471f27844ba10ba2c087cdd3440f966d061bf91a3e72dcdfd7587304ad

Reply:
Shipped separately https://github.com/rapid7/metasploit-framework/pull/15675